src/store/getters.ts

```diff
@@ -99,6 +99,7 @@ export default {
   grid: (state: ApplicationState, getters: Getters): number[][] =>
     state.gridVariables.map((variable: VariableWithVariants) =>
       getters.gridAssessments.map((assessment: Assessment) => {
+        if (state.variantCounts.length === 0) return NaN
```

Comment:
is null taken?

Reply:
No, i just liked the idea of not swapping datatype. Not a number felt a lot like 'it is a number, but i just don't know which one at the moment'

Do you think using null is better?